src/platform/Tizen/WiFiManager.cpp

```diff
@@ -750,6 +753,37 @@ CHIP_ERROR WiFiManager::RemoveAllConfigs(void)
     return err;
 }
 
+CHIP_ERROR WiFiManager::GetDeviceMACAddress(uint8_t * macAddress, size_t macAddressLen)
+{
+    VerifyOrReturnError(macAddress != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
+    VerifyOrReturnError(macAddressLen >= 6, CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
Minor: check length != 0 as well?

Reply:
What do you mean by that? You would like to be able to call `GetDeviceMACAddress` with `macAddressLen` equal to zero, like this: `GetDeviceMACAddress(&buffer, 0)`? Type size_t is unsigned, so length >= 6 implies length != 0.